tinygrad/tensor.py

```diff
@@ -124,7 +124,7 @@ def ones(cls, *shape, **kwargs): return cls(np.ones(shape, dtype=np.float32), **
   def empty(cls, *shape, **kwargs): return cls(np.empty(shape, dtype=np.float32), **kwargs)
 
   @classmethod
-  def randn(cls, *shape, **kwargs): return cls(np.random.default_rng().standard_normal(size=shape, dtype=np.float32), **kwargs)
+  def randn(cls, *shape, **kwargs): return cls(np.random.standard_normal(size=shape).astype(np.float32), **kwargs)
```

Comment:
This is slow, because it's first creating float64 then converting. Is there a way to set the seed with the other thing? Maybe Tensor has a long lived np.random.default_rng()?